<launch>
    <!-- vim: set ft=xml noet : -->
    <!-- example launch script for PX4 based FCU's -->

    <arg name="only_command" default="true" />
    <arg name="is_multicopter" default="false" />

    <group ns="rover1">
        <node pkg="offboard_control" name="Rover_offboard" type="offboard_control_node" output="screen">
            <param name ="is_multicopter" value= "$(arg is_multicopter)" />
            <param name="att_mode_rate" value="40"/>
            <param name="pva_mode_rate" value="20"/>
            <param name="landing_z" value="0.11"/>
            <param name="takeoff_z" value="1.4"/>
            <param name="takeoff_Kp" value="1.0" />
            <param name="landing_Kp" value="1.0" />

            <!-- Position Control parameters -->
            <param name="mass" value="1.535"/>       <!-- kg -->
            <param name="gz" value="9.81"/>          <!-- m/s^2 -->
            <param name="thrustRatio" value="1.7"/>  <!-- maxThrust/Weight -->
            <param name="kpx" value="10"/>
            <param name="kpy" value="10"/>
            <param name="kpz" value="10"/>
            <param name="kvx" value="5"/>
            <param name="kvy" value="5"/>
            <param name="kvz" value="7.5"/>
            <param name="kix" value="0"/>
            <param name="kiy" value="0"/>
            <param name="kiz" value="0"/>
            <param name="maxInteg_x" value="0"/>
            <param name="maxInteg_y" value="0"/>
            <param name="maxInteg_z" value="0"/>
        </node>
    </group>
</launch>